beacon-chain/cache/attestation.go

```diff
@@ -0,0 +1,234 @@
+package cache
+
+import (
+	"sync"
+
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v5/beacon-chain/operations/attestations/forkchoice"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	"github.com/prysmaticlabs/prysm/v5/crypto/bls"
+	ethpb "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1/attestation"
+	log "github.com/sirupsen/logrus"
+)
+
+type attGroup struct {
```

Comment:
In this file, all the methods have 0 documentation along with tests. If we want to know what exactly it does we have to read through everything line by line. Also we should have unit tests for all these methods